reference/extensions/authorization_plugins.rst

```diff
@@ -0,0 +1,66 @@
+.. _reference_extensions_authorization_plugin:
+
```

Comment:
This is missing the ``experimental`` notice. 

It can be included, check how other pages are doing it.